config/dependency.go

```diff
@@ -169,33 +171,49 @@ func getDependencyBlockConfigPathsByFilepath(configPath string, terragruntOption
 // This routine will go through the process of obtaining the outputs using `terragrunt output` from the target config.
 func dependencyBlocksToCtyValue(dependencyConfigs []Dependency, terragruntOptions *options.TerragruntOptions) (*cty.Value, error) {
 	paths := []string{}
+
 	// dependencyMap is the top level map that maps dependency block names to the encoded version, which includes
 	// various attributes for accessing information about the target config (including the module outputs).
 	dependencyMap := map[string]cty.Value{}
+	lock := sync.Mutex{}
+	g, _ := errgroup.WithContext(context.Background())
```

Comment:
NIT: We like to use descriptive variable names. Can you rename this to `dependencyGroup` or `dependencyErrGroup`?

Reply:
done!